sklearn/preprocessing/_encoders.py

```diff
@@ -796,6 +815,9 @@ def inverse_transform(self, X):
         """
         Convert the data back to the original representation.
 
+        If a categorical feature uses `None` *and* `np.nan` as a missing
+        value indicator, the inverse_transform will map `np.nan` to `np.nan`.
```

Comment:
The current wording (using "and") suggests that the sentence only applies if `X` contains **both** nans and Nones. However I think the following will be clearer:

```suggestion
        Nans will be inverse-transformed to Nans. Since both None and NaNs
        are mapped to NaN during `tranform`, this means that
        `inverse_transform(transform([[None], [NaN])) == [[NaN], [NaN]]`
```

Reply:
Actually my suggested sentence isn't even correct due to the bug noted in https://github.com/scikit-learn/scikit-learn/pull/19069#pullrequestreview-574162532.

I still think this sentence should be improved, which we can do once the bug is fixed